tools/code_check.sh

```diff
@@ -156,3 +156,4 @@ if [ $xmlout -eq 1 ]; then
 elif [ $QUICK_CHECK -eq 0 ]; then
   echo $CPPLINT_FILES | xargs python tools/cpplint.py 2>&1
 fi
+echo ""
```

Comment:
@senceryazici What does this addition do?

Reply:
Since it is definitely harmless, I'm going to go ahead and merge, but if you can explain I'd appreciate it.

Reply:
While testing on my local with 'act' (a tool for running GitHub actions on your local) the code check script constantly failed for some reason. Just for debugging I added a line to the end to see the result and it started passing.

Honestly (just like you've said) since it's totally harmless I didn't spend extra time on it's reason and let it be that way. But I'll definitely take a look at it when I have available time.